Document and support new default local stylesheet, ejecting theme style sheet (#741)
bullet-train-co/bullet_train-core: bullet_train/docs/stylesheets.md

# Style Sheets

> Bullet Train's stock UI theme, called “Light”, is built to use `tailwindcss` extensively.

Reviewer: Should we link to https://github.com/bullet-train-co/bullet_train-core/tree/main/bullet_train-themes-light and https://github.com/bullet-train-co/bullet_train-core/tree/main/bullet_train-themes-tailwind_css and give a note about how the Light theme is built on the tailwindcss theme?

Author: I've clarified the opening paragraph a bit, but I wouldn't point to the gems here, or away to the Themes doc until the question "but how do I modify these files" pops up as they read the Modify Theme Partials section just below, which points to the Theme doc.

https://github.com/bullet-train-co/bullet_train-core/blob/c8ece1807012351f99980f785f12729f67d772a0/bullet_train/docs/stylesheets.md?plain=1#L1-L2

Reviewer: Perfect!

As such, there are two ways to update the styling of your app: either by modifying theme partials, or by adding your own custom CSS.

## Modify Theme Partials

> Since `tailwindcss` is used, most style changes are done by ejecting theme partials into a local copy, and modifying the Tailwind classes within their HTML.

Reviewer: I think we can clarify this a little bit.

```suggestion
Since `tailwindcss` is used, most style changes are done by ejecting theme partials into your app's `app/views` directory, and modifying the Tailwind classes within the `.html.erb` templates.
```

You can eject only the theme files you wish to override or you can eject the whole UI theme for customization. You can find more information in the [indirection documentation](indirection) about using `bin/resolve` to find the theme partials to eject. Or see the [themes documentation](themes) for details on using the "Light" UI theme as a starting point for creating your own.

## Add custom CSS 

> If you need to add your own custom CSS, override the custom non-tailwind classes found in the `light` theme (custom classes are still used sparingly), or to import CSS from a third-party component, simply add to the `app/assets/stylesheets/application.custom.css` file.

Reviewer: I think we can clean this paragraph a little bit, but to be honest I don't know quite what it's saying.

```suggestion
To add your own custom CSS, override the custom non-tailwind classes found in the `light` theme (custom classes are still used sparingly), or to import CSS from a third-party component, add to the `app/assets/stylesheets/application.custom.css` file.
```

Author: Clarified the bottom paragraph also:

https://github.com/bullet-train-co/bullet_train-core/blob/6b60e20d709e657f4c8cb50d276aecd2c6c7efc7/bullet_train/docs/stylesheets.md?plain=1#L12-L14
